CONTRIBUTING.md

```diff
@@ -19,6 +19,16 @@ This creates an isolated virtual Python environment ("virtualenv") just for
 use with Sygnal, then installs Sygnal along with its dependencies, and lastly
 installs a handful of useful tools
 
+If you get `ConnectTimeoutError`, this is caused by slow internet whereby `pip` has a 
+default time out of _15 sec_. Run the following command by specifying timeout to be more
+than 15 sec.
+
+```bash
+python3 -m venv venv
+./venv/bin/pip timeout==1000 install -e '.[dev]'
```

Comment:
I think `timeout==1000` should instead be `--timeout 1000`, right?

And maybe lower the value to 60 or 120, at your discretion.

If pip doesn't receive anything in over a minute it's probably a sign that something is wrong elsewhere. Want to be mindful of striking a balance between allowing enough time and detecting errors. E.g., if PyPI is broken, it's probably not right for people to wait nearly 17 minutes (timeout 1000) for things to fail.

Reply:
> If pip doesn't receive anything in over a minute it's probably a sign that something is wrong elsewhere. Want to be mindful of striking a balance between allowing enough time and detecting errors. E.g., if PyPI is broken, it's probably not right for people to wait nearly 17 minutes (timeout 1000) for things to fail.

Thank you, I will have `timeout` to 60 sec

> I think timeout==1000 should instead be --timeout 1000, right?

I will update `timeout` without the `equal sign` just seen that on `pip --help` command